app/src/util/env.ts

```diff
@@ -0,0 +1,10 @@
+import Config from 'react-native-config';
+
+export const getEnvVariable = (envVar: string): string => {
+  if (envVar in Config && typeof Config[envVar] !== 'undefined') {
+    return Config[envVar] as string;
+  } else {
+    console.error(`Missing ${envVar} in .env`);
```

Comment:
Thoughts on throwing an error instead? This is fatal correct?
```ts
throw new Error(`Missing ${envVar} in .env`);
```

Reply:
Throwing an error is better 👍 . Without an envVar the app will crash or not work anyways